.gitignore

```diff
@@ -5,5 +5,10 @@ pom.xml
 pom.xml.asc
 *.jar
 *.class
-/.lein-*
-/.nrepl-port
+.lein-*
+.nrepl-port
+lib
```

Comment:
:) yes lib is not needed . it is my j2ee project need . You are a very strict person. verygood